virtualizarr/xarray.py

```diff
@@ -1,16 +1,259 @@
+from __future__ import annotations
+
+import os
 from collections.abc import Iterable, Mapping
+from concurrent.futures import Executor
+from pathlib import Path
 from typing import (
+    TYPE_CHECKING,
     Any,
+    Callable,
```

Comment:
Instead, import `Callable`, `MutableMapping`, and `Sequence` from `collections.abc` above.